frontend/components/suggestion-and-search.tsx

```diff
@@ -54,18 +82,21 @@ export default function SuggestionAndSearch({
     }
 
     const transformed = suggestions.map(
-      ({ title, link, example, preview, documentation }) => ({
+      ({ title, link, example, preview }) => ({
         title,
         link,
-        example,
-        preview,
+        example: {
+          ...example,
+          queryParams: example.queryParams || {},
+        },
+        preview: preview || {},
         isBadgeSuggestion: true,
-        documentation,
       })
     )
 
     return (
       <BadgeExamples
+        areBadgeSuggestions
```

Comment:
I take it this results in a `false` or some other falsy value being specified?

Reply:
The opposite; Including it results in `true`. This is a convention JSX inherited from HTML.